pkg/db/loader/issue_types_internal.go

```diff
@@ -18,7 +18,8 @@ type Result struct {
 }
 
 type Match struct {
-	Bug bugsv1.Bug `json:"bugInfo"`
+	Bug     bugsv1.Bug `json:"bugInfo"`
+	Context []string   `json:"context"`
```

Comment:
Can you add godoc that explains what a context is?

Reply:
I did some digging, this shows that the context should be the list of matches: https://github.com/openshift/ci-search/blob/30244af43eb695ff156d2d27b9a509a117156017/cmd/search/httpsearch.go#L165. I simply used it without finding the source. Now we know. I will document this.